types/express/index.d.ts

```diff
@@ -10,7 +10,6 @@
 
 import * as bodyParser from "body-parser";
 import * as core from "express-serve-static-core";
-import * as qs from "qs";
```

Comment:
Is `qs` used anywhere else? If not, it can be removed as a dependency.

Reply:
it turned out that it was only used for this feature, which is now removed, @jakebailey

Reply:
Great, then please remove it from package.json.

Reply:
Yeah, I think I should remove that dependency too